integrations/aws_linux.md

```diff
@@ -11,119 +11,119 @@ This page provides an overview of what you can do with the Amazon Linux AMI Host
-* Install the Wavefront proxy and Fluentd agent on the same host or on separate hosts.
-
-### Install the Wavefront Proxy
-
-The Wavefront proxy is required to send logs from your systems into Tanzu Observability. If you have not already done so, install a Wavefront proxy (version 11.4 or later).
-
-
-### Install Fluentd
-Install Fluentd to send logs data to Wavefront proxy.
-
-* See the [prerequisites](https://docs.fluentd.org/installation/before-install) for installing Fluentd.
-{% raw %}
-* To install the stable distribution package of Fluentd, called ```td-agent```, use one the following cURL commands, depending on the Linux distribution that you use:
-    
+## Linux Host Integration
+
+Monitoring Linux hosts is easy with Wavefront. This integration installs and configures Telegraf to send host metrics
+into Wavefront. Telegraf is a light-weight server process capable of collecting, processing, aggregating, and sending metrics to a [Wavefront proxy](https://docs.wavefront.com/proxies.html).
+
+In addition to setting up the metrics flow, this integration also installs a dashboard. Here's the CPU section of a dashboard displaying Linux host metrics.
+
+{% include image.md src="images/linux_db.png" width="80" %}
+
+
+## Linux Logs Setup
+
+This integration uses Fluentd input plugins and offers several setup options. For Fluentd installation prerequisites, see the [Fluentd Requirements](https://docs.fluentd.org/installation/before-install).
+
+* Use a proxy already running in your environment (preferred) or install a new proxy. 
+* Install the Wavefront proxy and Fluentd agent on the same host or on separate hosts.
+
+### Install the Wavefront Proxy
+
+The Wavefront proxy is required to send logs from your systems into Tanzu Observability. If you have not already done so, install a Wavefront proxy (version 11.4 or later).
+
+
+### Install Fluentd
+Install Fluentd to send logs data to Wavefront proxy.
+
+* See the [prerequisites](https://docs.fluentd.org/installation/before-install) for installing Fluentd.
+
{% raw %}
+* To install the stable distribution package of Fluentd, called ```td-agent```, use one the following cURL commands, depending on the Linux distribution that you use:
+    
    ```
-{% endraw %}
-    # Ubuntu 20.04 Focal
-    curl -L https://toolbelt.treasuredata.com/sh/install-ubuntu-focal-td-agent4.sh | sh
-
-   # Debian Bullseye
-   curl -L https://toolbelt.treasuredata.com//sh/install-debian-bullseye-td-agent4.sh | sh
-
-   # Redhat/CentOS 7/8
-   curl -L https://toolbelt.treasuredata.com/sh/install-redhat-td-agent4.sh | sh{% raw %}
+{% endraw %}
+    # Ubuntu 20.04 Focal
+    curl -L https://toolbelt.treasuredata.com/sh/install-ubuntu-focal-td-agent4.sh | sh
+
+   # Debian Bullseye
+   curl -L https://toolbelt.treasuredata.com//sh/install-debian-bullseye-td-agent4.sh | sh
+
+   # Redhat/CentOS 7/8
+   curl -L https://toolbelt.treasuredata.com/sh/install-redhat-td-agent4.sh | sh
{% raw %}
```

Comment:
There is a red ^M.. just want to confirm that this is expected :)

Reply:
Comes from the UI.